sm2a/Makefile

```diff
@@ -10,11 +10,11 @@ info_message = \
 
 count_down = \
 	@echo "Spinning up the system please wait..."; \
-	secs=40 ;\
-	while [ $$secs -gt 0 ]; do \
-		printf "%d\033[0K\r" $$secs; \
+	secs=40; \
+	while [ $secs -gt 0 ]; do \
+		printf "%d\033[0K\r" "$$secs"; \
 		sleep 1; \
-		: $$((secs--)); \
+		secs=$$((secs - 1)); \
```

Comment:
I am getting `/bin/sh: line 0: [: ecs: integer expression expected` when trying this (using macOS and zsh shell)